### PR TITLE
c8d/integration-cli: Skip part of TestListDanglingImagesWithDigests

### PR DESCRIPTION
**Pull request by vvoland**
- alternative to: https://github.com/moby/moby/pull/46856

**- What I did**
Skip part of TestListDanglingImagesWithDigests which tests graphdriver implementation specific behavior of `docker images --filter dangling=true`.

**- How I did it**

**- How to verify it**
Check TestListDanglingImagesWithDigests test result

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: integration-cli/docker_cli_by_digest_test.go [hunk 1]
```diff
@@ -341,6 +341,13 @@ func (s *DockerRegistrySuite) TestListDanglingImagesWithDigests(c *testing.T) {
 	// make sure repo shown, tag=<none>, digest = $digest1
 	re1 := regexp.MustCompile(`\s*` + repoName + `\s*<none>\s*` + digest1.String() + `\s`)
```
Comment (thaJeztah) [inline integration-cli/docker_cli_by_digest_test.go:342]: Is this case already covered by TestListImagesWithDigests above?

If it is, it may be cleaner to `t.Skip()` the whole test

Comment (thaJeztah) [reply integration-cli/docker_cli_by_digest_test.go:342]: Maybe this one test is actually multiple sub-tests 🤔 we could make sub-tests and skip those, but I guess we'd have to (temporarily) disable the "integration-cli is deprecated" check.

File: integration-cli/docker_cli_by_digest_test.go [hunk 2]
```diff
@@ -341,6 +344,7 @@ func (s *DockerRegistrySuite) TestListDanglingImagesWithDigests(c *testing.T) {
...
 	assert.Assert(c, re1.MatchString(out), "expected %q: %s", re1.String(), out)
+
```
Comment (thaJeztah) [inline integration-cli/docker_cli_by_digest_test.go:347]: oh! unrelated change (if you still have the branch open locally)

Comment (vvoland) [reply integration-cli/docker_cli_by_digest_test.go]: Fixed, sorry! 😄

**Review by thaJeztah — APPROVED** [review]
LGTM

File: integration-cli/docker_cli_by_digest_test.go (hunk 2 above)
Comment (thaJeztah) [reply integration-cli/docker_cli_by_digest_test.go]: No worries! Thought you still had the branch at hand, so would be easy to fix (and wouldn't have been a massive disaster if it was left)

**Review by dmcgowan — APPROVED** [review]
